src/config/delegates.json

```diff
@@ -1,5 +1,18 @@
 {
-  "0xd42Bb7FE32cDf68045f49553c6f851fD2c58B6a9": {
+  },
+  "0x31cd90C2788f3e390d2Bb72871f5aD3F1a4B22a1": {
+    "name": "LuukDAO",
+    "address": "0x31cd90C2788f3e390d2Bb72871f5aD3F1a4B22a1",
+    "logoUri": "https://uploads-ssl.webflow.com/6606afba60355d9881684ca4/66740edc0bcc48e50fa21eb1_-8ZYbBPK_400x400.jpg",
```

Comment:
For security, this app restricts where it loads images from. Can you please add your image directly here under `public/logos/delegatees` instead of linking to another domain